projects/smart-table/src/lib/smart-table/smart-table.component.html

```diff
@@ -42,12 +42,14 @@
               <ul class="a-list a-list--unstyled">
                 <li class="m-search-filter__results-item a-list__item"
                   *ngFor="let column of selectableColumns; index as i">
+                  <div *ngIf="column.canHide !== false">
```

Comment:
dit was nog niet genoeg om de code te doen werken, er is blijkbaar ook nog een wijziging aan initColumns nodig in de component.ts file. Graag testen in de example app dat de kolom effectief niet in de dropdown verschijnt.